utils/checks/snapshots/features.yaml

```diff
@@ -2197,6 +2197,10 @@
 - name: pallet-grandpa 4.0.0-dev
   features:
     - std
+- name: pallet-humanode-offences 0.1.0
+  features:
+    - default
```

Comment:
I wonder why `default` shows up here...

Reply:
Must be because it itself is a workspace target, so when it gets selected for tests it is included with `default` on.

Reply:
In the same way as for other pallets in our code base.

Reply:
Yes; we might want to advance this further so it shows features per compilation target.